test.yml

```diff
@@ -0,0 +1,470 @@
+              - ","
+              - !ImportValue "vpc-dev-PrivateSubnetIds-Group0-v2"
+
+    LoadBalancerSecurityGroupHttp:
+        
+        Metadata: {'Stackjack': {'AllowDestroy': true } }
+        Type: AWS::EC2::SecurityGroup
+        Properties:
+            VpcId: !ImportValue "vpc-dev-VpcId"
+            GroupDescription: Access to the load balancer that sits in front of ECS
+            SecurityGroupIngress:
+              - CidrIp: 10.0.0.0/8
+                IpProtocol: tcp
+                FromPort: 443
+                ToPort: 443
+              - CidrIp: 10.0.0.0/8
+                IpProtocol: tcp
+                FromPort: 80
+                ToPort: 80
+
+    ECSHostSecurityGroupIngressHttp:
+        
+        Metadata: {'Stackjack': {'AllowDestroy': true } }
+        Type: AWS::EC2::SecurityGroupIngress
+        Properties:
+            Description: Allow ALB to reach ECS for klaxon-dev/Http
+            SourceSecurityGroupId: !Ref LoadBalancerSecurityGroupHttp
+            IpProtocol: tcp
+            FromPort: 32768
+            ToPort: 65535
+            GroupId: !Sub "${ClusterHostSecurityGroup}"
+
+    LoadBalancerListenerHttp:
+        
+        Metadata: {'Stackjack': {'AllowDestroy': true } }
+        Type: AWS::ElasticLoadBalancingV2::Listener
+        DependsOn:
+          - LoadBalancerHttp
+          - DefaultTargetGroupHttp
+        Properties:
+            LoadBalancerArn: !Ref LoadBalancerHttp
+            Port: 443
+            Protocol: HTTPS
+            Certificates:
+              - CertificateArn: arn:aws:acm:us-east-1:912288704264:certificate/c1ff8358-9b24-4e68-9d38-28caf63c0fde
+            SslPolicy: ELBSecurityPolicy-TLS-1-2-Ext-2018-06
+            DefaultActions:
+              - Type: forward
+                TargetGroupArn: !Ref DefaultTargetGroupHttp
+
+    LoadBalancerListenerHttpRedirect:
+        
+        Metadata: {'Stackjack': {'AllowDestroy': true } }
+        Type: AWS::ElasticLoadBalancingV2::Listener
+        DependsOn:
+          - LoadBalancerHttp
+          - DefaultTargetGroupHttp
+        Properties:
+            LoadBalancerArn: !Ref LoadBalancerHttp
+            Port: 80
+            Protocol: HTTP
+            DefaultActions:
+              - Type: redirect
+                RedirectConfig:
+                    Host: "#{host}"
+                    Path: "/#{path}"
+                    Port: "443"
+                    Protocol: "HTTPS"
+                    Query: "#{query}"
+                    StatusCode: HTTP_301
+
+    DefaultTargetGroupHttp:
+        
+        Metadata: {'Stackjack': {'AllowDestroy': true } }
+        Type: AWS::ElasticLoadBalancingV2::TargetGroup
+        Properties:
+
+            VpcId: !ImportValue "vpc-dev-VpcId"
+            Port: 3001
+            Protocol: HTTP
+            HealthCheckPath: /healthcheck
+            Matcher: {'HttpCode': '200-299'}
+            HealthCheckIntervalSeconds: 10
+            HealthyThresholdCount: 2
```

Comment:
🤔 If we can manage to set `HealthCheckProtocol` to `HTTPS` as detailed in [this documentation](
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html#cfn-elasticloadbalancingv2-targetgroup-healthcheckprotocol), I think we will avoid the redirect.

Reply:
yes! this is exactly the hope